components/feature/prompts/src/main/res/layout/mozac_feature_promps_widget_month_picker.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<merge xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Do these need to be separate layouts?

Reply:
I removed the extra layout file!

At the benign didn't want to make the parent of `MonthAndYearPicker` a child of `ScrollView` because my intention wasn't creating a new type of `ScrollView`, I just wanted to create a new widget that was able to scroll in case there wasn't enough space. I guess custom view didn't compose very well as you are forced to extend a view and this view will be the parent view in the layout hierarchy.